Source/Scene/Model.js

```diff
@@ -2943,9 +2946,6 @@ define([
                                 BoundingSphere.clone(command.boundingVolume, pickCommand.boundingVolume);
                             }
                         }
-                    } else {
```

Comment:
Why did we delete this?  We're actually going to want this for KHR_materials_common and when we support glTF cameras.

CC @tfili